### DISCUSSION TITLE
Just few centimeters available. Do I even have a chance?

### DISCUSSION THREAD

**Discussion by eigenphase** [jomjol/AI-on-the-edge-device#3233] [Q&A]
I am struggling with my water meter which is in a box under the ground on the street. The gap to the cement cover is just a few centimeters.

I have played around a bit with duct tape (which is super hard because the cover is heavy) and rotating the lense by a bit but I just don't get a good image.

![IMG_6637](https://github.com/user-attachments/assets/8735b312-e8f1-4a3d-9343-3c42ff9befb9)

![IMG_6638](https://github.com/user-attachments/assets/fe1bf8f7-3d81-49a1-a891-ad1a2eb2c7a1)

<img width="988" alt="Screenshot 2024-09-12 at 14 34 03" src="https://github.com/user-attachments/assets/88f300d1-2489-49d2-8283-1a2cdeb5cc05">

Now even if I would be able to bring the counter on the camera, I realize that the cover is not completely fixed and just stepping on it moves it by a bit and with it, the image.

Do I even have a chance with this setup?

**Comment by nwalk7800** [comment]
You can keep turning the lens counter clockwise to get a closer focus distance.  From my experience the closest they can focus is about 4cm.  

There are also wide angle OV2640's available, but I don't know if that would be any better.

You could also gain about a cm if you remove the serial adapter and wire up 5v directly to the pins.
